packages/manager/modules/pci/src/projects/project/storages/instance-backups/instance-backup/delete/delete.html

```diff
@@ -20,4 +20,11 @@
             <span data-translate="pci_projects_project_storages_volume-instances_volume-instance_delete_error_not_deletable"></span>
         </oui-message>
     </div>
+    <div data-ng-if="$ctrl.associatedInstances.length > 0">
```

Comment:
You can apply the ngIf directive directly on the `<oui-message>` component I guess.

Reply:
done ty